src/lib/utils.py

```diff
@@ -21,6 +21,7 @@
 import datetime as dt
 import json
 import logging
+import magic
```

Comment:
3rd party imports should go after standard library imports (into the 2nd imports list).

Reply:
Should I sort the imports?

![grafik](https://user-images.githubusercontent.com/3226457/212492243-7d795308-8c1b-4bf4-9ee0-b34fa886f848.png)

Reply:
No, we don't use automatic sorter currently. Just put `magic` to the end of 3rd party modules.